zeroconf/test_asyncio.py

```diff
@@ -127,8 +126,10 @@ async def test_async_service_registration_name_conflict() -> None:
         "ash-3.local.",
         addresses=[socket.inet_aton("10.0.1.3")],
     )
+
     with pytest.raises(NonUniqueNameException):
-        await aiozc.async_register_service(conflicting_info)
+        task = await aiozc.async_register_service(conflicting_info)
+        await task
```

Comment:
@bdraco This and similar changes (adding a variable for what's being awaited) don't seem like they're necessary or related to the task at hand?

Reply:
This one is a bit odd because its an Coroutine that returns an Awaitable and python doesn't allow

`await await aiozc.async_register_service(conflicting_info)`

In production we don't want to await the awaitable (aka `task`) that it returns since we are ready to move on and start services and let it run in the backgroud.